src/components/SubscribeButton.tsx

```diff
@@ -17,37 +20,39 @@ type SubscribeButtonProps = {
 
 const ANIMATION_DURATION = 700
 
-const SubscribeButton: FC<SubscribeButtonProps> = ({ door, post, className }) => {
+const SubscribeButton: FC<SubscribeButtonProps> = ({
+  door,
+  post,
+  className
+}) => {
   const { data: subscriptions } = useSubscriptions()
   const { mutate: createSubscription } = useCreateSubscription()
   const { mutate: deleteSubscription } = useDeleteSubscription()
 
   const animationControl = useAnimation()
   const [animating, setAnimating] = useState(false)
 
-  const subscription = find(subscriptions, door ? { door } : { postUuid: post?.uuid })
+  const subscription = find(
+    subscriptions,
+    door ? { door } : { postUuid: post?.uuid }
+  )
```

Comment:
Er kanskje et case for å mappe subscription status om til en lookup heller enn å søke gjennom sånn. Kanskje.